src/rs_quark_assets/App.vue

```diff
@@ -54,6 +63,8 @@ const auth = useAuthentication()
 const p = useProviders()
 const route = useRoute()
 const sessionTimeout = useSessionTimeout()
+const installPromptEvent = ref()
+const eventTriggered = ref("undefined")
```

Comment:
```suggestion
const eventTriggered = ref()
```

Having a String with the value "undefined" is confusing

Reply:
I only did it because I was getting a warn. 
> [Vue warn]: Invalid prop: type check failed for prop "eventTriggered". Expected String with value "undefined", got Undefined